packages/orbit-components/src/Select/__tests__/Select.test.js

```diff
@@ -26,56 +30,54 @@ const dataAttrs = {
 };
 
 describe("Select", () => {
-  const component = shallow(
-    <Select
-      id={id}
-      value="1"
-      name={name}
-      placeholder={placeholder}
-      options={objectOptions}
-      onChange={mockChange}
-      tabIndex={tabIndex}
-      dataTest={dataTest}
-      readOnly
-      spaceAfter={spaceAfter}
-      dataAttrs={dataAttrs}
-    />,
-  );
+  it("should have expected DOM output", () => {
+    render(
+      <Select
+        id={id}
+        label={label}
+        value="1"
+        name={name}
+        placeholder={placeholder}
+        options={objectOptions}
+        onChange={onChange}
+        tabIndex={tabIndex}
+        dataTest={dataTest}
+        onFocus={onFocus}
+        onBlur={onBlur}
+        readOnly
+        spaceAfter={spaceAfter}
+        dataAttrs={dataAttrs}
+      />,
+    );
 
-  const select = component.find("Select__StyledSelect");
-  const label = component.find("Select__Label");
-  it("should have data-test", () => {
-    expect(select.render().prop("data-test")).toBe(dataTest);
-  });
-  it("should have data-state", () => {
-    expect(select.render().prop("data-state")).toBe("ok");
-    expect(select.render().prop("data-recording-ignore")).toBe("true");
-  });
-  it("should have name", () => {
-    expect(select.render().prop("attribs").name).toBe(name);
-  });
-  it("should have spaceAfter", () => {
-    expect(label.prop("spaceAfter")).toBe(spaceAfter);
-  });
-  it("should have tabindex", () => {
-    expect(select.render().prop("tabindex")).toBe(tabIndex);
-  });
-  it("should have placeholder", () => {
-    expect(select.childAt(0).text()).toBe(placeholder);
-    expect(select.render().prop("id")).toBe(id);
+    const select = screen.getByRole("combobox");
+
+    expect(screen.getByTestId(dataTest)).toBeInTheDocument();
+    expect(select).toHaveAttribute("id", id);
+    expect(select).toHaveAttribute("readonly");
+    expect(select).toHaveAttribute("tabindex", "-1");
+    expect(select).toHaveAttribute("data-state", "ok");
+    expect(select).toHaveAttribute("data-recording-ignore");
+    expect(select).toHaveAttribute("name", name);
+    expect(screen.getByLabelText(label)).toBeInTheDocument();
+    expect(screen.getByText(placeholder)).toBeInTheDocument();
+    expect(document.querySelector("svg")).toBeInTheDocument();
```

Comment:
No need to assert that the icon exists, there's no logic involved in its visibility (no condition), and it's not a critical Select behavior.
```suggestion
```